test/onnx/test_operators.py

```diff
@@ -15,6 +15,7 @@
 import os
 import shutil
 import torch.testing._internal.common_utils as common
+import numpy as np
```

Comment:
Better not add ONNX support in this PR adding the function - it just adds complexity.